python/tests/conftest.py

```diff
@@ -229,6 +229,31 @@ def s3_tmpdir() -> str:
         logging.warn("Could not delete directory: %s", s3fs_path)
 
 
+@pytest.fixture(scope="session")
```

Comment:
too slow to use the default test SparkSession, we need to do some cleanup in another pull request

Reply:
Could you clarify why it is slow to use SparkSession? where is the part which takes the most of the time?

Reply:
`rikai_spark()` is the default SparkSession before https://github.com/eto-ai/rikai/pull/369

I could revert this part and just use `spark()` since I have tested it locally.

`spark()` is slow because of extra dependencies and mlflow initialization. It takes minutes for me to download jars from maven.